src/osd/scrubber/scrub_backend.cc

```diff
@@ -40,24 +40,22 @@ std::ostream& ScrubBackend::logger_prefix(std::ostream* out,
 // ////////////////////////////////////////////////////////////////////////// //
 
 // for a Primary
-ScrubBackend::ScrubBackend(PgScrubber& scrubber,
-                           PGBackend& backend,
-                           PG& pg,
+ScrubBackend::ScrubBackend(ScrubBeListener& scrubber,
+                           PgScrubBeListener& pg,
```

Comment:
Why are these separate?

Reply:
The services supplied by the PgScrubber vs. direct requests from the PG (that have no equivalent in the PgScrubber).